docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+https://github.com/hazelcast/hazelcast/pull/23535[#23535], https://github.com/hazelcast/hazelcast/pull/23954[#23954]
+* Improved the nested loop joins algorithm by making the `CreateDagVisitor` interface generic.
+https://github.com/hazelcast/hazelcast/pull/23368[#23368]
+* Implemented the `TO_CHAR` function which is capable of formatting dates, timestamps, and numbers.
+https://github.com/hazelcast/hazelcast/pull/22529[#22529]
+* Implemented `JSON=VARCHAR` comparison.
+https://github.com/hazelcast/hazelcast/pull/22746[#22746]
+
+=== Jet Engine
+
+* Each Jet engine job was creating a client for connecting to the remote cluster, resulting in an excessive number of clients.
+This has been fixed by introducing a single data connection which can be reused for all job instances.
+https://github.com/hazelcast/hazelcast/pull/23472[#23472], https://github.com/hazelcast/hazelcast/pull/23849[#23849]
+* Added MongoDB read (source) and write (sink) processors.
+https://github.com/hazelcast/hazelcast/pull/23348[#23348]
+* Introduced `JobStatusListener` as an alternative to retrieve a job status via the `Job.getStatus()` method.
+https://github.com/hazelcast/hazelcast/pull/23193[#23193]
+* Updated the https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc/com/hazelcast/jet/Job.html#isUserCancelled--[job API] to add the ability
+to distinguish the user-cancelled jobs from the failed ones.
+https://github.com/hazelcast/hazelcast/pull/22924[#22924]
+* Added `flock` to guard all the concurrent `pip` executions (upgrading `pip` and `protobuf` versions) in the Jet-to-Python script.
+https://github.com/hazelcast/hazelcast/pull/22829[#22829]
+
+=== Security
+
+* Added the TLS configuration property `keyMaterialDuration` which allows using a configurable timeout for caching keyStores and trustStores.
+See xref:security:tls-configuration.adoc[TLS Configuration] for the details on how to configure this property.
+https://github.com/hazelcast/hazelcast/pull/23307[#23307]
+
+=== Serialization
+
+* Hazelcast now allows serializing unsupported types via xref:serialization:compact-serialization.adoc#using-compact-serialization-with-zero-configuration[zero-configuration]
+provided that there is an explicit serializer implemented for the unsupported type.
+https://github.com/hazelcast/hazelcast/pull/23449[#23449]
+* Setting a generic record now fails if the value is not of the same type of generic record.
+https://github.com/hazelcast/hazelcast/pull/23522[#23522]
+* Introduced clearer exception messages for Generic MapStore, instead of `UndefinedErrorCodeException`.
+https://github.com/hazelcast/hazelcast/pull/22765[#22765]
+
+=== WAN Replication
+
+* Added the REST endpoint `/hazelcast/rest/wan/sync/progress` which allows tracking the WAN synchronization progress.
+https://github.com/hazelcast/hazelcast/pull/22958[#22958]
+
+=== Connectors
+
+* Added support of predicates in DML (Data Manipulation Language) in JDBC SQL connector.
```

Comment:
Preserve order of mapping with implicit field order definition for JDBC (https://github.com/hazelcast/hazelcast/pull/22760) is worth mentioning

Reply:
also community contribution to Kafka connector: https://github.com/hazelcast/hazelcast/pull/21546